src/state/selection/selectors/basic.ts

```diff
@@ -13,3 +13,5 @@ export const getSelectedAgentMetadata = (state: State) =>
     state.selection.selectedAgentMetadata;
 export const getSelectedUIDisplayData = (state: State) =>
     state.selection.selectedUIDisplayData;
+export const getCurrentColorSettings = (state: State) =>
+    state.selection.currentColorSettings;
```

Comment:
should also be singular (otherwise it sounds like it's going to be a list of colors, when it's just the singular binary setting)

Reply:
What about `currentColorSetting`/`getCurrentColorSetting` for state and `AgentColorMapping` for the typing.

Examples:
`if (colorSetting === AgentColorMapping.UserSelected)`
or
`currentColorSetting: AgentColorMapping.Default,`

Reply:
I wouldn't call it a Mapping because that generally means it's a lookup "table". but this is just the value you use in another lookup